### PR TITLE
docs: update installation instructions

### PR DESCRIPTION
**Pull request by salaxander**
Signed-off-by: Xander Grzywinski <xgrzywinski@microsoft.com>

Add homebrew and release download methods to installation docs

### CODE REVIEW

File: website/docs/installation.md [hunk 1]
```diff
@@ -2,26 +2,23 @@
 title: Installation
 ---
 
-The following instructions are for **Ubuntu 22.04** with the dependency versions supported as part of the [dev container](./contributing.md/#visual-studio-code-development-container) environment we use for builds and tests. For other distributions and OS, refer to the appropriate installation instructions for each of the components instead.
```
Comment (sozercan) [inline website/docs/installation.md:5]: can we move the existing docs under contributing/development?

File: website/docs/installation.md [hunk 2]
```diff
@@ -2,26 +2,23 @@
...
+
+```bash
+brew install copa
+```
+
+## GitHub
+You can download the latest and previous versions of `copa` from the [GitHub releases page](https://github.com/project-copacetic/copacetic/releases).
+
+## Build from source
+`copa` can also be built directly from source:
```
Comment (sozercan) [inline website/docs/installation.md:16]: do we want to keep the requirements here?

File: website/docs/installation.md [hunk 3]
```diff
@@ -2,26 +2,23 @@
...
-   sudo apt install git
-   git clone https://github.com/project-copacetic/copacetic
-   cd copacetic
-   make
-   # OPTIONAL: install copa to a pathed folder
-   sudo mv dist/linux_amd64/release/copa /usr/local/bin/
-   ```
\ No newline at end of file
+## Homebrew
+On MacOS and Linux, `copa` can be installed via [Homebrew](https://brew.sh/):
```
Comment (sozercan) [inline website/docs/installation.md:6]:
```suggestion
On macOS and Linux, `copa` can be installed via [Homebrew](https://brew.sh/):
```

File: website/docs/installation.md (hunk 2 above)
Comment (CodeMonkeyLeet) [reply website/docs/installation.md:16]: I think we just need to rationalize where this information shows up: My preference from other projects is that all build & test dependencies are specified in dev setup instructions (so docker and make should be mentioned here), as well as their minimum version requirements. Others like buildkit might be specified at point of use, such as in the [patching instructions](https://github.com/project-copacetic/copacetic/blob/main/docs/tutorials/patch.md).

**Review by CodeMonkeyLeet — COMMENTED** [review]
@salaxander @sozercan One additional meta-comment: Release automation relies on commit title prefixes to generate release notes, so please make sure your commits are correctly tagged per [contributor guidelines](https://github.com/project-copacetic/copacetic/blob/main/CONTRIBUTING.md#pull-requests). As maintainers that means:

- all commits are appropriately prefixed if using _rebase merge_
- title of PR that is used as the default commit title is appropriately prefixed if using _squash and merge_.

File: website/docs/installation.md (hunk 2 above)
Comment (salaxander) [reply website/docs/installation.md:16]: Maybe we just change that "Build from source" header to "Development Setup" and keep that section the same as it currently is in the docs?

Comment (CodeMonkeyLeet) [reply website/docs/installation.md:16]: Yeah, I think that makes sense

**Review by CodeMonkeyLeet — APPROVED** [review]

**Comment by CodeMonkeyLeet** [comment]
@salaxander while merging this I noticed that you have DCO signoffs under multiple email accounts; can you check your environments (your local git config in each clone/fork of this repo) to ensure that you are using the one recognized by github as your primary? (xgrzywinski@microsoft.com I think)

**Comment by salaxander** [comment]
@CodeMonkeyLeet Yeah my MSFT alias changed a while back and I just got round to updating it on GitHub and in my gitconfig during this PR. Should be good in the future!

**Pull request merged**